content/source/docs/cloud/api/subscriptions.html.md

```diff
@@ -0,0 +1,197 @@
+---
+layout: "cloud"
+page_title: "Subscriptions - API Docs - Terraform Cloud and Terraform Enterprise"
+---
+
+[200]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/200
+[201]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/201
+[202]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/202
+[204]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/204
+[400]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400
+[401]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/401
+[403]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/403
+[404]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404
+[409]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/409
+[412]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/412
+[422]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/422
+[429]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/429
+[500]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/500
+[504]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/504
+[JSON API document]: /docs/cloud/api/index.html#json-api-documents
+[JSON API error object]: http://jsonapi.org/format/#error-objects
+
+# Subscriptions API
+
+-> **Note:** The subscriptions API is only available in Terraform Cloud.
+
+An organization can subscribe to different [feature sets](./feature-sets.html), which represent the [pricing plans](/docs/cloud/paid.html) available in Terraform Cloud. An organization's [entitlement set](./index.html#feature-entitlements) is calculated using its subscription and feature set.
```

Comment:
```suggestion
An organization can subscribe to different [feature sets](./feature-sets.html), which represent the [pricing plans](/docs/cloud/paid.html) available in Terraform Cloud. An organization's [entitlement set](./index.html#feature-entitlements) is calculated using its subscription and feature set.

To change the subscription for an organization, use the billing settings in the Terraform Cloud UI. Subscriptions cannot be changed via the API. 
```

Figured we should just say it before someone asks.

Reply:
Excellent addition! Technically there is an API endpoint for updating subscriptions, and I was going back and forth a bit on whether to include it here - it's _almost_ useless when you're not using the UI because of some Stripe magic on the frontend.